### ISSUE TITLE
Connection error whilst running locally

### ISSUE THREAD

**Issue by simonarcher99** [hackcollective/wind-curtailment#70]
Hi I'm interested in contributing to this project but I'm struggling to run locally. I have done the following

- installed requirements
- ran `streamlit run main.py --server.port 8082`

I am getting the following error from the `get_data()` function:

> connection to server at "34.88.31.153", port 5432 failed: Operation timed out
        Is the server running on that host and accepting TCP/IP connections?
        
Am I doing something wrong? I'm happy to add further instructions to the readme if there's steps missing. Thanks

**Comment by peterdudfield** [comment]
Hi @simonarcher99. Thanks for getting in contact. 

Myself and @archydeberker just been chatting about this. Currently database is `whitelisted` so people wont have access to that immediately. This is hopefully understandable for security reasons. 

What we've done in the past is to setup a local sqlite database and add some data into there. We should add a sample of this to this repo, and then we first run the app locally, we could connect to that. 

if you wanted to do this, you would have setup sqlite database and pull X days of data in to there. Do you think you would be interested in doing this? No worries if your not

**Comment by simonarcher99** [comment]
Yeah that makes sense. I'd be interested in giving that a go. I'll try to get it set up and then have a look at writing a script in the repo that would create a database locally and fill it with some data.

**Comment by peterdudfield** [comment]
> Yeah that makes sense. I'd be interested in giving that a go. I'll try to get it set up and then have a look at writing a script in the repo that would create a database locally and fill it with some data.

Thanks, There's some sql in a folder that sets up the tables. And from memory if you use the `get data` script the tables are automatically made. But i'll let you check it>

Get in contact if you get stuck at all

**Comment by archydeberker** [comment]
I'll try and take a look at this tomorrow @simonarcher99 and see if we can figure out a good way to make a cloud SQL dump available to everybody

**Comment by simonarcher99** [comment]
Hi @archydeberker thanks for having a look, let me know if there's anything I can do to help out. I had a go at saving to an sqlite db locally but it looks like the sequel queries aren't compatible with sqlite
